<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>viewstdrecord</class>
 <widget class="QMainWindow" name="viewstdrecord">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1335</width>
    <height>1029</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>View Student Record</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/new/images/icons/logoicon.ico</normaloff>:/new/images/icons/logoicon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget {
background-image: url(:/new/images/Images/a72082c00f056ac229e10ffcbdac9059.jpg);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_15">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="">
       <layout class="QVBoxLayout" name="verticalLayout_13">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_18">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Search for Student&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
            </property>
            <property name="text">
             <string>Search:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="searchBox">
            <property name="styleSheet">
             <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_19">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Sort List&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
            </property>
            <property name="text">
             <string>Search By</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="sortBy">
            <property name="styleSheet">
             <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
            </property>
            <item>
             <property name="text">
              <string>firstname</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>course</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>enrolledYear</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>lastname</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>GPA</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>email</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Search">
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_20">
            <item>
             <widget class="QPushButton" name="Back">
              <property name="text">
               <string>Back to Staff Panel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="showresult">
              <property name="text">
               <string>Show Result</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_21">
            <item>
             <widget class="QPushButton" name="Previous">
              <property name="text">
               <string>Previous</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="Next">
              <property name="text">
               <string>Next</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="Student_photo">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_14">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="styleSheet">
         <string notr="true">font: 16pt &quot;MS UI Gothic&quot;;</string>
        </property>
        <property name="title">
         <string>Name</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label">
               <property name="styleSheet">
                <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color:white;</string>
               </property>
               <property name="text">
                <string>Registration Number</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="registration"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="styleSheet">
                  <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color:white;</string>
                 </property>
                 <property name="text">
                  <string>First Name</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="student_frstname">
                 <property name="styleSheet">
                  <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="styleSheet">
                  <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color:white;</string>
                 </property>
                 <property name="text">
                  <string>Middle Name</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="Student_lastname">
                 <property name="styleSheet">
                  <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="styleSheet">
                  <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color:white;</string>
                 </property>
                 <property name="text">
                  <string>Last Name</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="Student_midname">
                 <property name="styleSheet">
                  <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_22">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="styleSheet">
           <string notr="true">font: 16pt &quot;MS UI Gothic&quot;;</string>
          </property>
          <property name="title">
           <string>Address</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="styleSheet">
                <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color:white;</string>
               </property>
               <property name="text">
                <string>Permanent Address:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="Student_permanentAdd">
               <property name="styleSheet">
                <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="styleSheet">
                <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color:white;</string>
               </property>
               <property name="text">
                <string>Temporary Address:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="Student_TemporaryAdd">
               <property name="styleSheet">
                <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="styleSheet">
           <string notr="true">font: 16pt &quot;MS UI Gothic&quot;;</string>
          </property>
          <property name="title">
           <string>Sex</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QRadioButton" name="Stu_male">
             <property name="styleSheet">
              <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color:white;</string>
             </property>
             <property name="text">
              <string>Male</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="Stu_female">
             <property name="styleSheet">
              <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color:white;</string>
             </property>
             <property name="text">
              <string>Female</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="Stu_others">
             <property name="styleSheet">
              <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color:white;</string>
             </property>
             <property name="text">
              <string>Others</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="styleSheet">
           <string notr="true">font: 16pt &quot;MS UI Gothic&quot;;</string>
          </property>
          <property name="title">
           <string>Contact</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="styleSheet">
                <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color:white;</string>
               </property>
               <property name="text">
                <string>Email:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="Student_email">
               <property name="styleSheet">
                <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="styleSheet">
                <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color:white;</string>
               </property>
               <property name="text">
                <string>Phone No:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="Student_phone">
               <property name="styleSheet">
                <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_19">
        <item>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="styleSheet">
           <string notr="true">font: 16pt &quot;MS UI Gothic&quot;;</string>
          </property>
          <property name="title">
           <string>Family Information</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_11">
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="styleSheet">
                  <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color:white;</string>
                 </property>
                 <property name="text">
                  <string>Father's Name:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="Student_fathername">
                 <property name="styleSheet">
                  <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_12">
               <item>
                <widget class="QLabel" name="label_10">
                 <property name="styleSheet">
                  <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color:white;</string>
                 </property>
                 <property name="text">
                  <string>Phone No:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="Student_fatherphone">
                 <property name="styleSheet">
                  <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_11">
                 <property name="styleSheet">
                  <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color:white;</string>
                 </property>
                 <property name="text">
                  <string>Occupation:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="Student_FatherOccupation">
                 <property name="styleSheet">
                  <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_13">
               <item>
                <widget class="QLabel" name="label_12">
                 <property name="styleSheet">
                  <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color:white;</string>
                 </property>
                 <property name="text">
                  <string>Mother's Name:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="Student_Mothername">
                 <property name="styleSheet">
                  <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_14">
               <item>
                <widget class="QLabel" name="label_13">
                 <property name="styleSheet">
                  <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color:white;</string>
                 </property>
                 <property name="text">
                  <string>Phone No:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="Student_MotherPhone">
                 <property name="styleSheet">
                  <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_14">
                 <property name="styleSheet">
                  <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color:white;</string>
                 </property>
                 <property name="text">
                  <string>Occupation:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="Student_motheroccupation">
                 <property name="styleSheet">
                  <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_15">
               <item>
                <widget class="QLabel" name="label_15">
                 <property name="styleSheet">
                  <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color:white;</string>
                 </property>
                 <property name="text">
                  <string>Guardian's Name:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="Student_GuardianName">
                 <property name="styleSheet">
                  <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_16">
               <item>
                <widget class="QLabel" name="label_16">
                 <property name="styleSheet">
                  <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color:white;</string>
                 </property>
                 <property name="text">
                  <string>Phone No:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="Student_guardianphone">
                 <property name="styleSheet">
                  <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_17">
                 <property name="styleSheet">
                  <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color:white;</string>
                 </property>
                 <property name="text">
                  <string>Occupation:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="Student_guardianoccupation">
                 <property name="styleSheet">
                  <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_6">
          <property name="styleSheet">
           <string notr="true">font: 16pt &quot;MS UI Gothic&quot;;</string>
          </property>
          <property name="title">
           <string>Status</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_17">
             <item>
              <widget class="QLabel" name="label_27">
               <property name="styleSheet">
                <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color:white;</string>
               </property>
               <property name="text">
                <string>Enrolled Year:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="Student_enrolledeyear">
               <property name="styleSheet">
                <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_18">
             <item>
              <widget class="QLabel" name="label_28">
               <property name="styleSheet">
                <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color:white;</string>
               </property>
               <property name="text">
                <string>Course:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="Student_Course">
               <property name="styleSheet">
                <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_7">
          <property name="styleSheet">
           <string notr="true">font: 16pt &quot;MS UI Gothic&quot;;</string>
          </property>
          <property name="title">
           <string>Description</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <widget class="QTextEdit" name="Student_Description">
             <property name="styleSheet">
              <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>sortBy</tabstop>
  <tabstop>Next</tabstop>
  <tabstop>Search</tabstop>
  <tabstop>Previous</tabstop>
  <tabstop>searchBox</tabstop>
  <tabstop>Back</tabstop>
  <tabstop>registration</tabstop>
  <tabstop>Student_permanentAdd</tabstop>
  <tabstop>Student_TemporaryAdd</tabstop>
  <tabstop>Stu_male</tabstop>
  <tabstop>Stu_female</tabstop>
  <tabstop>Stu_others</tabstop>
  <tabstop>Student_email</tabstop>
  <tabstop>Student_phone</tabstop>
  <tabstop>Student_fathername</tabstop>
  <tabstop>Student_fatherphone</tabstop>
  <tabstop>Student_FatherOccupation</tabstop>
  <tabstop>Student_Mothername</tabstop>
  <tabstop>Student_MotherPhone</tabstop>
  <tabstop>Student_motheroccupation</tabstop>
  <tabstop>Student_GuardianName</tabstop>
  <tabstop>Student_guardianphone</tabstop>
  <tabstop>Student_guardianoccupation</tabstop>
  <tabstop>Student_enrolledeyear</tabstop>
  <tabstop>Student_Course</tabstop>
  <tabstop>Student_Description</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
